### PR TITLE
Onboarding Mobile styles

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/121?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #113** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/113?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #114** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/114?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #115** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/115?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #116** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/116?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #118** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/118?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #121** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/121?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
            * **PR #119** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/119?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/121?utm_source=stack-comment).

File: packages/app/features/onboarding/screen.tsx [hunk 1]
```diff
@@ -279,41 +247,6 @@ function SendAccountCongratulations() {
           </Paragraph>
           <IconCopy />
         </XStack>
-
-        {__DEV__ && !!sendAcct && (
```
[the remark this answers is not in the document]
Comment (0xBigBoss) [reply packages/app/features/onboarding/screen.tsx:273]: Still dropping it?

[the remark this answers is not in the document]
Comment (youngkidwarrior) [reply packages/app/features/onboarding/screen.tsx:273]: Fixed

File: packages/app/features/auth/onboarding/screen.tsx
Comment (0xBigBoss) [inline packages/app/features/auth/onboarding/screen.tsx:1]: why are there 2 onboarding screens?

File: packages/contracts/.gitignore
Comment (0xBigBoss) [inline packages/contracts/.gitignore:1]: sry we actually need these.

File: supabase/database-generated.types.ts
Comment (0xBigBoss) [inline supabase/database-generated.types.ts:1]: nice, i think i introduced this when I upgraded the supabase CLI.

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Feb 28, 12:46 AM EST**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/121).

**Pull request merged**